src/display/editor/stamp.js

```diff
@@ -536,6 +545,10 @@ class StampEditor extends AnnotationEditor {
       }
     });
     this.#observer.observe(this.div);
+    this._uiManager._signal.addEventListener("abort", () => {
+      this.#observer.disconnect();
+      this.#observer = null;
+    });
```

Comment:
Do we want to remove this listener as well?
```suggestion
    }, { once: true });
```